packages/react/src/components/dateInput/components/datePicker/DatePicker.module.scss

```diff
@@ -187,6 +187,11 @@
     justify-content: flex-end;
     transform: translateY(-3px);
 
+    [aria-disabled="true"] {
+      color: rgba(16, 16, 16, 0.3);
```

Comment:
Suggestion: would it be possible to use some lighter variant of black?

Reply:
- [ ] Apply HDS black variant variable